src/TypeMirror.spec.lua

```diff
@@ -0,0 +1,64 @@
+return function()
+	local Type = require(script.Parent.Type)
+	local Mirror = require(script.Parent.TypeMirror)
+	local allowedTypes = {
+		Type.Binding,
+		Type.Element,
+		Type.HostChangeEvent,
+		Type.HostEvent,
+		Type.StatefulComponentClass,
+		Type.StatefulComponentInstance,
+		Type.VirtualTree
+	}
```

Comment:
This is interesting redundancy. Do you think there's any risk that we'll forget to update this table, which would allow tests to pass erroneously?

Reply:
That's a good point - TypeMirror should return a strict table with this list instead